types/index.test.ts

```diff
@@ -77,6 +77,60 @@ const cookieFunction: AzureFunction = async function (context: Context) {
     };
 };
 
+const httpResponseObjectFunction: AzureFunction = async function (context: Context) {
+    context.res = context.res as HttpResponseObject;
+    context.res = {
+        body: {
+            hello: 'world',
+        },
+        status: 200,
+        statusCode: 200,
+        headers: {
+            'content-type': 'application/json',
+        },
+        cookies: [
+            {
+                name: 'cookiename',
+                value: 'cookievalue',
+                expires: Date.now(),
+            },
+        ],
+        enableContentNegotiation: false,
+    };
+};
+
+const statusStringFunction: AzureFunction = async function (context: Context) {
+    context.res = context.res as HttpResponseObject;
+    context.res = {
+        status: '200',
+        statusCode: '200',
+    };
+};
+
+const httpResponseApiFunction: AzureFunction = async function (context: Context) {
+    context.res = context.res as HttpResponseApi;
+    context.res.status(200);
+    context.res.setHeader('hello', 'world');
+    context.res.set('hello', 'world');
+    context.res.header('hello', 'world');
+    const hello: string = context.res.get('hello');
+    const hello2: string = context.res.getHeader('hello');
+    context.res.removeHeader('hello');
+    context.res.type('application/json');
+    context.res.body = {
+        hello,
+        hello2
```

Comment:
Are you using VS Code for your local development? For me at least, it auto-fixes lint errors like this one (where it wants a comma after hello2) when I save the file. If that's not happening for you, we might want to double check the repo settings in `.vscode`. Also double check you have the recommended extensions installed (specifically the one for eslint)

Reply:
Yeah good callout. I am using VSCode and I have the extensions installed but I didn't have the npm package `eslint-plugin-deprecation`, so when you added that it broke ESLint for me. Running `npm i` fixed the issue, so I fixed the linting errors too